src/Client.php

```diff
@@ -178,6 +179,17 @@ public function cart($cartID = false)
     }
 
     /**
+     *  Get a order
+     *
+     *  @param string|false $orderID
+     *  @return Moltin\Entities\Order
+     */
+    public function order($orderID = false)
```

Comment:
`$orderId` isn't used in this method anywhere. It's the correct thing to do insomuch as it will allow us to use the following:

```
$items = $moltin->order('0cba9ee5-25c3-4671-b9ea-b76f7375328c')->items();
```

Instead of the proposed syntax:

```
$order = $moltin->order();
$order->setID('0cba9ee5-25c3-4671-b9ea-b76f7375328c');
$order->items();
```

This then follows the same pattern as `carts` which makes them much easier to work with.